### DISCUSSION TITLE
Stitches styled does not work in ssr NextJS

### DISCUSSION THREAD

**Discussion by Masoty** [stitchesjs/stitches#1163] [Help]
I encountered the problem that stitches do not work in ssr NextJS in the program router, the global styles work, but the styles declared in styled do not work, how can I fix this?

**Comment by danielboso** [comment]
> I encountered the problem that stitches do not work in ssr NextJS in the program router, the global styles work, but the styles declared in styled do not work, how can I fix this?

Have you tried this? https://nextjs.org/docs/app/building-your-application/styling/css-in-js#configuring-css-in-js-in-app
